core/src/main/scala/kafka/server/BrokerServer.scala

```diff
@@ -401,6 +404,19 @@ class BrokerServer(
         ))
       val fetchManager = new FetchManager(Time.SYSTEM, new FetchSessionCache(fetchSessionCacheShards))
 
+      val shareFetchSessionCache : ShareSessionCache = new ShareSessionCache(
+        config.shareGroupMaxGroups * config.shareGroupMaxSize,
+        KafkaServer.MIN_INCREMENTAL_FETCH_SESSION_EVICTION_MS)
+      val sharePartitionManager = new SharePartitionManager(
+        replicaManager,
+        Time.SYSTEM,
```

Comment:
Time is already defined in the class, can we please re-use that:
```suggestion
        time,
```